core/src/main/scripts/importer/allowed_data_types.txt

```diff
@@ -32,3 +32,5 @@ METHYLATION	CONTINUOUS	methylation_hm450
 FUSION	FUSION	fusion
 PROTEIN_LEVEL	LOG2-VALUE	rppa
 PROTEIN_LEVEL	Z-SCORE	rppa_Zscores
+PROTEIN_LEVEL	CONTINUOUS	protein_quantification
+PROTEIN_LEVEL	Z-SCORE	protein_quantification_zscores
```

Comment:
I'll add this to list of issues with the current CPTAC data, so we can consider changing it when we update the data. To me it makes more sense to keep it lower case, because 
1. some other terms are also lower which usually are in upper case (e.g. rna and rrpa)
2. it looks like this whole column should be lower case
3. bit of a hassle to go through the pipeline to regenerate this data only for 1 small change.